### PR TITLE
[Speed] Updates Auto Minify references

### PR DESCRIPTION
**Pull request by angelampcosta**

### CODE REVIEW

File: content/analytics/web-analytics/understanding-web-analytics/page-load-time-summary.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Below is a list of all the components you can inspect:
...
-- **Processing** (`domComplete` - `domInteractive`): How long it took to render the page. This includes loading any resources that block page rendering, including images, scripts, and style sheets. If this number is big, optimize your document architecture, resource size, or configure settings in the Cloudflare Speed app, such as Auto Minify the source code. This document process can be drilled down more with `domInteractive`, `domContentLoadedEventStart`, `domContentLoadedEventEnd`, and `domComplete`.
+- **Processing** (`domComplete` - `domInteractive`): How long it took to render the page. This includes loading any resources that block page rendering, including images, scripts, and style sheets. If this number is big, optimize your document architecture, resource size, or configure settings in the Cloudflare Speed app, such as Auto Minify (deprecated) the source code. This document process can be drilled down more with `domInteractive`, `domContentLoadedEventStart`, `domContentLoadedEventEnd`, and `domComplete`.
```
Comment (ngayerie) [inline content/analytics/web-analytics/understanding-web-analytics/page-load-time-summary.md]: I would remove the reference to Auto Minify entirely in this section

File: content/pages/configuration/debugging-pages.md [hunk 2]
```diff
@@ -74,9 +74,9 @@ If you are not using Functions or have reviewed that your Functions configuratio
 
 ## Differences between `pages.dev` and custom domains
 
-If your custom domain is proxied (orange-clouded) through Cloudflare, your zone's settings such as Auto Minify and caching will apply.
+If your custom domain is proxied (orange-clouded) through Cloudflare, your zone's settings such as Auto Minify (deprecated) and caching will apply.
```
Comment (ngayerie) [inline content/pages/configuration/debugging-pages.md]: I would remove the reference to Auto Minify entirely in this section
Maybe `, your zone's settings (like caching) will apply.` ?

**Review by GregBrimble — APPROVED** [review]

File: content/pages/configuration/debugging-pages.md (hunk 2 above)
Comment (angelampcosta) [reply content/pages/configuration/debugging-pages.md]: @ngayerie can I also remove the following paragraph `If you are experiencing issues with a framework, like Nuxt.js, only on the custom domain, review if Auto Minify (deprecated) is enabled (log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login) > **Speed** > **Optimization** > **Content Optimization** > **Auto Minify**) for HTML and disable it.`?

Comment (ngayerie) [reply content/pages/configuration/debugging-pages.md]: this one I would leave for now :)

**Review by ngayerie — APPROVED** [review]

**Review by pedrosousa — APPROVED** [review]

**Pull request merged**